src/integration/java/com/ibm/eventstreams/connect/mqsource/MQSourceTaskIT.java

```diff
@@ -654,4 +657,39 @@ public void verifyEmptyTextMessage() throws Exception {
 
         connectTask.commitRecord(kafkaMessage);
     }
+
+    @Test
+    public void testJmsWorkerWithCustomReciveForConsumerAndCustomReconnectValues() throws Exception {
+        connectTask = getSourceTaskWithEmptyKafkaOffset();
+
+        final Map<String, String> connectorConfigProps = createExactlyOnceConnectorProperties();
+        connectorConfigProps.put("mq.message.body.jms", "true");
+        connectorConfigProps.put("mq.record.builder",
+                "com.ibm.eventstreams.connect.mqsource.builders.DefaultRecordBuilder");
+        connectorConfigProps.put("mq.message.receive.timeout", "5000");
```

Comment:
```suggestion
        connectorConfigProps.put("mq.receive.timeout", "5000");
```

Is this the intended config?